config/locales/views/en.facility_notifications.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  views:
+    facility_notifications:
+      in_review:
+        head: "Orders In Review"
+        no_orders: "No orders are currently in review"
+        instructions: |
+          The orders listed below are still in review by the customer.  The time the review window will close is listed.
+          You may manually mark an order as reviewed, allowing you to create a statement for or journal the order immediately.
+        submit: "Mark As Reviewed"
+      index:
```

Comment:
Actually, should this file have been included in this PR? I don't see any changes to the referencing views.